docs/products/compute/compute-instances/guides/local-disk-encryption/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Local Disk Encryption"
+description: Local disk encryption helps you to protect the information stored on your Linode's disk. This guide shows how to implement local disk encryption.
+keywords: ['local disk encryption','disk encryption', 'encryption','security']
+aliases: ['/guides/local-disk-encryption/']
+tags: ["security","encryption"]
+modified: 2024-07-01
+modified_by:
+  name: Akamai
+published: 2024-07-01
+title: Local Disk Encryption
+authors: ["Akamai"]
+---
+
+Local disk encryption ensures that your data stored on compute instances is encrypted. Disk encryption protects against data leakage if the disk is ever removed from the datacenter, recycled or disposed of. Systems within the datacenter manage the encryption and decryption for you.
```

Comment:
Serial comma after recycled? Not mandatory, but a definite Akamai writing pet peeve. :-) (Actually, I don't know if Linode writing guideline enforce the serial comma like we do.)

Reply:
Added the comma.